pyang/__init__.py

```diff
@@ -1,4 +1,4 @@
 """The pyang library for parsing, validating, and converting YANG modules"""
 
-__version__ = '2.5.0'
-__date__ = '2021-06-21'
+__version__ = '2.5.0.m1'
+__date__ = '2021-07-08'
```

Comment:
You should not modify this file. The version is modified only before release.

Reply:
Please disregard this file changes. They are relevant only for my private fork.

Reply:
But if you want to merge this PR, this file change will be merged.

Reply:
Reverted to master version.